src/components/ForecastSummary.js

```diff
@@ -1,45 +0,0 @@
-import React from "react";
-import PropTypes from "prop-types";
-import WeatherIcon from "react-icons-weather";
-import "../styles/ForecastSummary.css";
-
-function ForecastSummary(props) {
-  const { date, description, icon, temperature, onSelect } = props;
-  const formattedDate = new Date(date).toDateString();
-  return (
-    <div className="forecast-summary" data-testid="forecast-summary">
```

Comment:
Each nested div in your jsx should be indented in one level like this
`<div>
  <div>
    <p>hi</p>
  </div>
</div>`

This makes it much easier to see what elements are in which container